jobs/JWAVE_PRDGEN_GRIDDED

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+
+date
+export PS4=' $SECONDS + '
+set -xa
+
+# PATH for working directory
+export NET=${NET:-gfs}
+export RUN=${RUN:-gfs}
+export COMPONENT=${COMPONENT:-wave}
+export RUNwave=${RUNwave:-${RUN}${COMPONENT}}
+export envir=${envir:-ops}
+export fstart=${fstart:-0}
+export FHMAX_WAV=${FHMAX_WAV:-180}       #180 Total of hours to process
+export FHMAX_HF_WAV=${FHMAX_HF_WAV:-72}  #from 00 to  72 inc=3
+export FHOUT_WAV=${FHOUT_WAV:-6}         #from 72 to 180 inc=6
+export FHOUT_HF_WAV=${FHOUT_HF_WAV:-3}
+export maxtries=720
+
+
+export HOMEwave=${HOMEwave:-$(dirname $(dirname $0))} # parent directory of current job card
+export DATA=${DATA:-${DATAROOT:?}/${job}.$$}
+mkdir -p $DATA
+cd $DATA
+
+cyc=${cyc:-00}
+export cycle=${cycle:-t${cyc}z}
+export pgmout=OUTPUT.$$
+#export modID=${RUN}${NET}
+export modID=${modID:-${NET}${COMPONENT}}
+
+####################################
+# Path to HOME Directory
+####################################
+
+export CODEwave=${CODEwave:-${NWROOT}/${COMPONENT}_code.${wave_code_ver}/${code_pkg}}
+
+export EXECwave=${EXECwave:-HOMEwave/exec}
+export FIXwave=${FIXwave:-HOMEwave/fix}
+export PARMwave=${PARMwave:-HOMEwave/parm}
+export USHwave=${USHwave:-HOMEwave/ush}
+export EXECcode=${EXECcode:-CODEwave/exec}
+
+# Set PDY
+setpdy.sh
+. PDY
+####################################
+# load modules
+####################################
+module use ${HOMEwave}/modulefiles
```

Comment:
Same comment as in jobs/JWAVE_GEMPAK

Reply:
@RobertoPadilla-NOAA Please remove the "module use ${HOMEwave}/modulefiles" line from both jobs/JWAVE_PRDGEN_GRIDDEN and jobs/JWAVE_GEMPAK. The modulefiles will be loaded from the workflow manager level (ecflow ecf scripts or jobs/rocoto/*.sh scripts). I will be adding the new rocoto scripts (jobs/rocoto) for this.

Also, don't forget to rename those scripts per my comment above. Thanks!

Reply:
Done